Gopkg.toml

```diff
@@ -50,33 +49,35 @@ required = [
   name = "gopkg.in/yaml.v2"
   version = "v2.2.1"
 
+# NOTE: if we get to update k8s.io to > 1.13, we can remove k8s-dynamic-fake-simple.patch in ./hack/update-deps.sh
```

Comment:
Let's create an issue to track the removal of the dynamic-fake-simple-patch and link it in here.